frontend/src/components/Searchbar/Searchbar.tsx

```diff
@@ -0,0 +1,89 @@
+import { KeyboardEvent, useCallback, useRef } from 'react'
+import { BiSearch } from 'react-icons/bi'
+import {
+  Box,
+  forwardRef,
+  Icon,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputProps,
+  useMergeRefs,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { SEARCHBAR_THEME_KEY } from '~/theme/components/Searchbar'
+
+import IconButton from '../IconButton'
+
+export interface SearchbarProps extends InputProps {
```

Comment:
Is it helpful to explicitly pass in the variant as one of the `SearchbarProps` and set a default value?